action.yml

```diff
@@ -43,11 +43,14 @@ inputs:
     description: 'Relative path from where you run Chromatic to your Storybook config directory'
     required: false
   only:
-    description: 'Only run a single story or a subset of stories'
+    description: 'Deprecated, use onlyStoryNames instead'
```

Comment:
Small note. Let's choose between Deprecated or [Deprecated] to keep it more uniform.